.gitignore

```diff
@@ -37,3 +37,5 @@ obj_dir/
 
 example.exe.dSYM/
 example.exe
+
+ipxact2systemverilog/ipxact2systemverilog.egg-info/
```

Comment:
Why egg format? Python egg format for python has been deprecated since 2012 https://peps.python.org/pep-0427/